code/modules/jobs/job_types/roguetown/garrison/mayor.dm

```diff
@@ -9,14 +9,15 @@
 	allowed_sexes = list(MALE, FEMALE)
 	allowed_races = RACES_TOLERATED_UP
 	allowed_ages = list(AGE_MIDDLEAGED, AGE_OLD)
-	tutorial = "Whether you are a crooked politician or a true benefactor, the cityfolk now turn to you for guidance on smaller matters. \
+	tutorial = "Whether you are a crooked politician or a true benefactor, you are the Mayor of the city of Rockhill and oversees both Lowtown and Hightown, the cityfolk now turn to you for guidance on smaller matters. \
 				The Duke may hold the official title, but with the Sheriff under your command, will you submit to the weight of tradition or reshape the very idea of authority?"
 	whitelist_req = TRUE
 	outfit = /datum/outfit/job/roguetown/mayor
 	display_order = JDO_MAYOR
 	min_pq = 5
 	max_pq = null
-	give_bank_account = 35
+	give_bank_account = 100
+	spells = list(SPELL_CONVERT_ROLE_GUARD)
```

Comment:
We probably don't want them to be inviting people into the guard, thats the sheriffs domain.